newIDE/app/src/stories/componentStories/MainFrameToolbar.stories.js

```diff
@@ -0,0 +1,157 @@
+// @flow
+import * as React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import muiDecorator from '../ThemeDecorator';
+import paperDecorator from '../PaperDecorator';
+
+import MainFrameToolbar, {
+  type MainFrameToolbarProps,
+  type ToolbarInterface,
+} from '../../MainFrame/Toolbar';
+import ToolbarIcon from '../../UI/ToolbarIcon';
+
+export default {
+  title: 'MainFrameToolbar',
+  component: MainFrameToolbar,
+  decorators: [paperDecorator, muiDecorator],
+};
+
+const fakeEditorToolbar = (
+  <span
+    style={{
+      flex: 1,
+      display: 'flex',
+      justifyContent: 'flex-end',
+    }}
+  >
+    <ToolbarIcon src="res/ribbon_default/bug32.png" tooltip="test" />
+    <ToolbarIcon src="res/ribbon_default/bug32.png" tooltip="test" />
+    <ToolbarIcon src="res/ribbon_default/bug32.png" tooltip="test" />
+    <ToolbarIcon src="res/ribbon_default/bug32.png" tooltip="test" />
+  </span>
+);
+
+const defaultProps: MainFrameToolbarProps = {
+  showProjectIcons: true,
+  hasProject: false,
+  toggleProjectManager: () => {},
+  requestUpdate: () => {},
+  simulateUpdateDownloaded: () => {},
+  simulateUpdateAvailable: () => {},
+  exportProject: () => {},
+  onPreviewWithoutHotReload: () => {},
+  onOpenDebugger: () => {},
+  onNetworkPreview: () => {},
+  onHotReloadPreview: () => {},
+  setPreviewOverride: () => {},
+  canDoNetworkPreview: true,
+  isPreviewEnabled: false,
+  hasPreviewsRunning: false,
+  previewState: {
+    isPreviewOverriden: false,
+    previewLayoutName: null,
+    previewExternalLayoutName: null,
+    overridenPreviewLayoutName: null,
+    overridenPreviewExternalLayoutName: null,
+  },
+  exportProject: () => {},
+  hasProject: false,
+};
+
+export const EditorNotLoaded = () => (
+  <MainFrameToolbar {...defaultProps} showProjectIcons={false} />
+);
+
+export const NoProjectLoaded = () => <MainFrameToolbar {...defaultProps} />;
+
+export const ProjectOpenOnScene = () => (
+  <MainFrameToolbar
+    {...defaultProps}
+    isPreviewEnabled
+    hasProject
+    previewState={{
+      isPreviewOverriden: false,
+      overridenPreviewExternalLayoutName: null,
+      overridenPreviewLayoutName: null,
+      previewExternalLayoutName: null,
+      previewLayoutName: 'test',
```

Comment:
Nitpicking: try to give names like "fake layout", "fake external layout".
This can be useful to see if there was a mistake in a component (out of my mind, as an example: if a story is showing "The scene fake external layout is launched" rather than "The scene test is launched")